src/API/Management/ClientGrants.php

```diff
@@ -2,75 +2,193 @@
 
 namespace Auth0\SDK\API\Management;
 
-use Auth0\SDK\API\Header\ContentType;
+use Auth0\SDK\Exception\CoreException;
 
+/**
+ * Class ClientGrants.
+ * Handles requests to the Client Grants endpoint of the v2 Management API.
+ *
+ * @package Auth0\SDK\API\Management
+ */
 class ClientGrants extends GenericResource
 {
+
     /**
+     * Get all Client Grants, by page if desired.
+     * Required scope: "read:client_grants"
+     *
+     * @param array        $params   Additional URL parameters to send:
+     *      - "audience" to filter be a specific API audience identifier.
+     *      - "client_id" to return an object.
+     *      - "include_totals" to return an object.
+     * @param null|integer $page     The page number, zero based.
+     * @param null|integer $per_page The amount of entries per page.
      *
-     * @param  string      $id
-     * @param  null|string $audience
      * @return mixed
+     *
+     * @throws \Exception Thrown by the Guzzle HTTP client when there is a problem with the API call.
+     *
+     * @link https://auth0.com/docs/api/management/v2#!/Client_Grants/get_client_grants
      */
-    public function get($id, $audience = null)
+    public function getAll(array $params = [], $page = null, $per_page = null)
```

Comment:
Looks like I'm changing the method name here but I'm not. The `get` method still exists (in a semi-broken state), moved to the bottom of the class with a `TODO` to deprecate.